gui/src/components/Balances.tsx

```diff
@@ -16,20 +16,24 @@ export function Balances() {
     "db_get_erc20_balances",
     { address }
   );
+  const { data: settings } = useInvoke<GeneralSettings>("settings_get");
+
+  const reorderedBalances = (balances || [])
+    .map<[`0x${string}`, bigint]>(([c, b]) => [c, BigInt(b)]) // is it possible to get the proper type from backend directly?
```

Comment:
It's weird that you must type it because it works on line 32, or do you see an error there too?

Reply:
btw, for the `0x{string}` you already have an `Address` type

Reply:
The types gets wrong without that cast (`(bigint | '0x${string}')[][]`), but here its not necessary to be fair. That's a residue from an old implementation. Ill remove it.